frontend/src/SignInPage.js

```diff
@@ -4,6 +4,7 @@ import PasswordField from './PasswordField'
 import { object, string } from 'yup'
 import {
```

Comment:
Looks like an '@chakra-ui/core' slipped through here.

Reply:
Yep, it was one of the merge conflicts, guess I got too occupied with the imports themselves.